integration/bmp/bms_cmd_test.go

```
@@ -22,7 +21,7 @@ var _ = Describe("`$bmp bms --deployment` integration tests", func() {
 
 	Describe("$bmp bms --deployment", func() {
 		BeforeEach(func() {
-			deployment := GetDeployment()
+			deployment, _ := GetDeployment()
```

Why are you ignoring the error? Please add a check for no error